CONTRIBUTING.md

```diff
@@ -52,10 +52,61 @@ Pull requests often need some real-world testing.
 
 2. Run `npm install`.
 
-3. Go to the `webpack-dev-server` module (`cd node_modules/webpack-dev-server`), and run `npm run prepublish`.
+3. Go to the `webpack-dev-server` module (`cd node_modules/webpack-dev-server`), and run `npm run prepare`.
 
 The pull request is now ready to be tested.
 
+**Test Commands**
+
+- `$ npm run test` : 
+This will run linting tests as pre-tests *(`npm run lint`)* commands and the `jest` tests
+
+- `$ npm run test:watch`
+This will runt the `jest` test in `watch` mode
+
+
+## Commit message
+
+### Commit Message Format
+
+Each commit message consists of a **header**, a **body** and a **footer**. The header has a special
+format that includes a **type** and a **subject**:
+
+```md
+<type>: <subject>
+<BLANK LINE>
+
+<body>
+<BLANK LINE>
+<footer>
+```
+
+This is the list of _type_ of commits that we accept:
+
+-   ast
+-   break
+-   chore
+-   cli
+-   docs
+-   feat
+-   fix
+-   misc
+-   tests
+
+The **header** is mandatory.
+
+Any line of the commit message cannot be longer 100 characters. This allows the message to be easier
+to read on GitHub as well as in several git tools.
+
+For more information about what each part of the template mean, head up to the documentation in the
+[angular repo](https://github.com/angular/angular.js/blob/master/CONTRIBUTING.md#commit-message-format)
```

Comment:
let's remove this from this issue and send other PR with this update, also we use convetial-commits, so type is invalid

Reply:
type in the commit format ?